src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java

```diff
@@ -731,6 +738,14 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
 				break;
 			case CLAN_SETTINGS_INFO_PAGE_WIDGET:
 				clientThread.invoke(() -> createSyncButton(CLAN_SETTINGS_INFO_PAGE_WIDGET_ID));
+				if (namechangesSubmitted)
```

Comment:
Why isn't the same thing done when the members page widget is loaded?

Reply:
The members page can't be opened without the clan settings page being opened first. So there is no reason to run it there as well. Clan settings -> Members page is the navigation route and we only want to run this once.